server/src/routes/foodEntry.js

```diff
@@ -11,9 +11,18 @@ const router = express.Router();
 // Guide: https://www.prisma.io/docs/concepts/components/prisma-client/crud
 
 router.post(
-  "/",
-  //validator.isUniqueName,
-  controller.createFoodEntry
+  '/',
+  validator.isFoodEntryId,
```

Comment:
isFoodEntryId does not make sense for creating a food entry since the id would not have been created yet